lib/services/local/configuration.go

```diff
@@ -194,6 +195,52 @@ func (s *ClusterConfigurationService) DeleteAuthPreference(ctx context.Context)
 	return nil
 }
 
+// GetPAMConfig gets types.PAMConfig from the backend.
+func (s *ClusterConfigurationService) GetPAMConfig(ctx context.Context) (types.PAMConfig, error) {
+	item, err := s.Get(ctx, backend.Key(clusterConfigPrefix, pamPrefix))
+	if err != nil {
+		if trace.IsNotFound(err) {
+			return nil, trace.NotFound("pam configuration not found")
+		}
+		return nil, trace.Wrap(err)
+	}
+	return services.UnmarshalPAMConfig(item.Value,
+		services.WithResourceID(item.ID), services.WithExpires(item.Expires))
```

Comment:
@fspmarshall What's your opinion on making skip validation the default here? I know it's a single resource, but it's also in the critical path of starting a session.

Reply:
Definitely in favor.  Not only is it slow, but having it in critical paths has previously caused bugs that could lead to teleport instances entering permanently bad states.  Skip it.

Reply:
Does this mean that schema validation should be removed?

Reply:
Disabled